src/main/kotlin/com/sourcegraph/cody/chat/ui/LlmDropdown.kt

```diff
@@ -59,8 +59,8 @@ class LlmDropdown(
             ?: availableModels.find { it.model == selectedFromHistory?.model }
 
     selectedItem =
-        if (selectedModel?.codyProOnly == true && isCurrentUserFree())
-            availableModels.find { it.default }
+        if (selectedModel?.isCodyProOnly() == true && isCurrentUserFree())
+            availableModels.getOrNull(0)
```

Comment:
Out of curiosity, if we do not have `default` anymore isn't there any tag which would replace it?
Like recommended LLM?

Reply:
I've just replaced it with the fallback convention of the [head of the list](https://github.com/sourcegraph/cody/pull/4780/files#diff-fe82b352761565cc5b6eb38a881a5bda7c83ad2c6b9231fa4bb25364154cc407R272-R277). The model structs are immutable having a mutable field tracking the user's selection was a bit tricky and also easy to get wrong (multiple default's was possible). And we always fell back to just grabbing the head of the list if we couldn't find one.

I could see adding a default tag, but I would think to only use it for what the server originally sent as the default, not as the user's current selection.